content/api/sending-domains.apib

```diff
@@ -353,7 +353,7 @@ Returns the full sending domain object, except the `domain` field.
 
 + Data Structure
     + tracking_domain (string) - Associated tracking domain. To remove the current value, set it to an empty string. The tracking domain and sending domain must belong to the same subaccount.
-    + dkim (object) - Object in which DKIM key configuration is defined. Will override the current object.
+    + dkim (object) - Object in which DKIM key configuration is defined. If DKIM configuration is specified in an update, the new DKIM information will be verified prior to updating the sending domain. If dkim verification fails, the update will return a failure.
```

Comment:
Since this is in the context of an update, we can assume that it will always be new if provided and save some words. Would you mind setting the description to:

"DKIM configuration. Will be verified prior to update. If verification fails, the update will fail." 

or similar?

Reply:
Will do. Thanks for the input.